_books/ion-1-1/src/binary/e_expressions.md

```diff
@@ -445,7 +476,7 @@ As noted in the table above:
 #### Expression groups
 
 This section describes the encoding of an expression group. For an explanation of what an expression group is and how to use it,
-see _[Expression groups](../todo.md)_.
+see [_Argument groups_](../macros/macros_by_example.md#argument-groups).
 
 An expression group begins with a [`FlexUInt`](primitives/flex_uint.md). If the `FlexUInt`'s value
 is:
```

Comment:
We should consolidate on either "Expression groups" or "Argument groups." I prefer 'expression groups', but don't feel strongly about it.

Reply:
Opened this, to avoid stepping on other possible in-flight changes: https://github.com/amazon-ion/ion-docs/issues/377